src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -67,21 +67,22 @@
         >
           {{ item.index + 1 }} of {{ item.siblingsCount }} symbols inside
         </span>
-        <Reference
+        <OptionalLink
           :id="item.uid"
           :url="item.path || ''"
```

Comment:
It looks like the `Reference` component already has logic to utilize a span when it isn't active and/or doesn't have a URL. Are you sure we're not duplicating logic by creating a new component?